runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -797,3 +771,47 @@ TR_J9SharedCache::getClassChainOffsetOfIdentifyingLoaderForClazzInSharedCache(TR
    uintptrj_t classChainOffsetInSharedCache = (uintptrj_t) offsetInSharedCacheFromPointer(classChainIdentifyingLoaderForClazz);
    return classChainOffsetInSharedCache;
    }
+
+void *
+TR_J9JITaaSServerSharedCache::pointerFromOffsetInSharedCache(void *offset)
+   {
```

Comment:
Maybe we should store the stream into `TR_J9JITaaSServerSharedCache` object so that we don't have to call a TLS routine every time.

Reply:
Will do